packages/flutter/test/rendering/editable_test.dart

```diff
@@ -422,4 +422,24 @@ void main() {
     expect(updatedSelection.extentOffset, 5);
     expect(selectionChangedCount, 1);
   });
+
+  test('editable hasFocus correctly initialized', () {
+    final TextSelectionDelegate delegate = FakeEditableTextState();
```

Comment:
Can you add `// Regression test for <issue link>` here?

Reply:
added